flair/nn.py

```diff
@@ -68,13 +70,13 @@ def save(self, model_file: Union[str, Path]):
         torch.save(model_state, str(model_file), pickle_protocol=4)
 
     @classmethod
-    def load(cls, model: Union[str, Path]):
+    def load(cls, model_file: Union[str, Path]):
```

Comment:
@yahshibu The `load()` method is also used to load a pre-trained tagger by passing a string, as in `SequenceTagger.load('ner')`. In this case, the string is not a model file, so I think the original variable name `model` might be better here.